include/gba/defines.h

```diff
@@ -11,7 +11,7 @@
 #define EWRAM_DATA __attribute__((section("__DATA,ewram_data")))
 #else
 #define IWRAM_DATA __attribute__((section("iwram_data")))
-#define EWRAM_DATA __attribute__((section("ewram_data")))
+#define EWRAM_DATA __attribute__((section(".sbss")))
```

Comment:
Why the section rename?

Reply:
I needed it to get the linkerscript to work with the RAM-code initialization that expansion has, for some reason: https://github.com/rh-hideout/pokeemerald-expansion/pull/3892

not entirely clear on why. If there's no need for that though I can revert it